Vagrantfile

```diff
@@ -6,72 +6,21 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
```

Comment:
Will do.

This is the Hasicorp/Vagrant recommended way of providing a multi-line shell script to the shell provisioner without having to create a separate shell script that Vagrant references (by convention it's usually called `bootstrap.sh`). It has the added benefit of putting the code you're executing in the same spot as what calls it, making it easier to figure out what's happening.